net-mgmt/zabbix-proxy/src/opnsense/mvc/app/models/OPNsense/Zabbixproxy/General.xml

```diff
@@ -79,5 +79,17 @@
             <default>4</default>
             <Required>N</Required>
         </timeout>
+        <encryption type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </encryption>
+        <encryptionid type="TextField">
+            <default></default>
+            <Required>N</Required>
+        </encryptionid>
+        <encryptionkey type="TextField">
+            <default></default>
+            <Required>N</Required>
```

Comment:
Can you add the correct validations to ID and PSK please? https://www.zabbix.com/documentation/3.0/manual/encryption/using_pre_shared_keys